apps/nextjs-website/src/components/atoms/UserInfo/UserInfo.tsx

```diff
@@ -28,20 +54,64 @@ const UserInfo: FC = () => {
       gap={1}
       justifyContent='flex-end'
     >
-      <Hidden smDown>
-        <PersonOutline />
-        <Typography variant='body2' sx={{ fontWeight: 600, fontSize: '14px' }}>
+      <Stack
+        direction='row'
+        alignItems='center'
+        justifyContent='flex-end'
+        onClick={handleClick}
+      >
+        <Avatar
+          sx={{
+            width: 20,
+            height: 20,
+            mr: 1,
+            bgcolor: palette.text.primary,
+          }}
+        ></Avatar>
+        <Typography variant='body2' sx={{ fontSize: 14, fontWeight: 600 }}>
           {user.attributes.given_name} {user.attributes.family_name}
         </Typography>
-      </Hidden>
-      <Button
-        size='small'
-        variant='contained'
-        onClick={signOut}
-        sx={{ height: '30px' }}
+        <IconButton size='small' sx={{ ml: 1 }}>
+          <ArrowDropDownOutlinedIcon
+            sx={{ width: 24, height: 24 }}
+          ></ArrowDropDownOutlinedIcon>
+        </IconButton>
+      </Stack>
+
+      <Menu
+        anchorEl={menu}
+        id='account-menu'
+        open={open}
+        onClose={handleClose}
+        onClick={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'right',
+        }}
+        sx={{
+          filter: 'drop-shadow(0px 2px 8px rgba(0,0,0,0.32))',
+          mt: 1.5,
+        }}
       >
-        {logout}
-      </Button>
+        <MenuItem onClick={handleClose} sx={{ p: 2 }}>
+          <Link
+            href='/profile/personal-data'
+            sx={{ textDecoration: 'none', color: palette.text.primary }}
+          >
+            {shared.yourData}
+          </Link>
+        </MenuItem>
+        <MenuItem onClick={signOut} sx={{ p: 2 }}>
+          {logout}
+          <ListItemIcon sx={{ ml: 1 }}>
+            <Logout fontSize='small' sx={{ color: palette.text.primary }} />
+          </ListItemIcon>
+        </MenuItem>
+      </Menu>
```

Comment:
There is an issue with this menu. It only works if i click on the label otherwise it close dropdown.
[screencast-localhost_3000-2023.11.22-17_19_18.webm](https://github.com/pagopa/developer-portal/assets/24475319/bcfe8a3e-65db-400e-82ea-7704a3678ff9)